### PR TITLE
feature: add managed fields in json schema for resources, such as create_time and update_time

### PR DESCRIPTION
**Pull request by nic-chen**
### What this PR does / why we need it:

add managed fields in json schema for resources, such as create_time and update_time

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

relate to #2433

### CODE REVIEW

**Comment by nic-chen** [comment]
add to milestone 2.0 because need these fields to work with new `manager api` #2444

File: t/admin/consumers.t [hunk 1]
```diff
@@ -322,3 +322,67 @@ GET /t
...
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/consumers',
+                ngx.HTTP_PUT,
+                [[{
+                     "username":"pony",
+                     "desc": "new consumer",
+                     "create_time": 1602883670,
```
Comment (juzhiyuan) [inline t/admin/consumers.t:338]: So this means the manager api will append create_time & update_time to those POST/PUT requests sent from frontend?

File: t/admin/routes.t [hunk 2]
```diff
@@ -2326,3 +2326,77 @@ GET /t
...
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, message = t('/apisix/admin/routes/1',
+                 ngx.HTTP_DELETE,
+                 nil,
+                 [[{
+                    "action": "delete"
+                }]]
```
Comment (juzhiyuan) [inline t/admin/routes.t:2392]: Codes format

Comment (nic-chen) [reply t/admin/routes.t:2392]: It's common used in APISIX.

File: t/admin/consumers.t (hunk 1 above)
Comment (nic-chen) [reply t/admin/consumers.t:338]: sure.

**Review by membphis — APPROVED** [review]
nice PR ^_^

**Review by juzhiyuan — APPROVED** [review]

**Comment by membphis** [comment]
@nic-chen I think you'd better add a link to the question in this PR. The community needs to know which issue is solved.

**Comment by nic-chen** [comment]
> @nic-chen I think you'd better add a link to the question in this PR. The community needs to know which issue is solved.

sure.I have modified the summary。

**Pull request merged**